floss/language/utils.py

```diff
@@ -465,6 +465,34 @@ def get_struct_string_candidates(pe: pefile.PE) -> Iterable[StructString]:
             # dozens of seconds or more (suspect many minutes).
 
 
+def get_raw_xrefs_rdata_i386(pe: pefile.PE, buf: bytes) -> Iterable[VA]:
+    """
+    scan for raw xrefs in .rdata section
```

Comment:
what are raw xrefs? can you add an example disassembly listing and add some comments, please?

Reply:
from the screenshots in https://github.com/mandiant/flare-floss/pull/885#issuecomment-1742411052 I don't see if these are strings and it would help to have some comments explaining what the algorithm looks for

Reply:
Raw xrefs refer to unprocessed xrefs in the binary file, indicating points where strings can be divided. I'll add an example with comments.

Reply:
Thanks, can you share a few example binary hashes?

Reply:
https://github.com/mandiant/flare-floss-testfiles/blob/master/language/rust/rust-hello/bin/rust-hello.exe